bugbug/repository.py

```diff
@@ -20,7 +20,14 @@
 import threading
 from datetime import datetime
 from functools import lru_cache
-from typing import Collection, Iterable, Iterator, NewType, Set, Union
+from typing import (
+    Collection,
+    Iterable,
+    Iterator,
+    NewType,
+    Set,
```

Comment:
In Python 3.9 and higher, we could use the standard collections as generic types instead of the capitalized ones (e.g., `set` instead of `Set`).[^1]

[^1]: https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
I think `Set` was in the existing code before I made any changes, however I will be sure to change any capitalized standard collections to the generic types!